lib/utils/fs.go

```diff
@@ -17,13 +17,22 @@ limitations under the License.
 package utils
 
 import (
+	"context"
+	"errors"
 	"os"
 	"path/filepath"
+	"time"
+
+	"github.com/gofrs/flock"
 
 	"github.com/gravitational/teleport"
 	"github.com/gravitational/trace"
 )
 
+// ErrUnsuccessfulLockTry designates an error when we temporarily couldn't acquire lock
+// (most probably it was already locked by someone else), another try might succeed.
+var ErrUnsuccessfulLockTry = errors.New("could not acquire lock on the file at this time")
```

Comment:
maybe return a `trace.Retry` error if the idea is to inform users they can retry?

Reply:
Hm, so something like `return nil, trace.Retry(ErrUnsuccessfulLockTry, "")`? Because we still need to provide error to retry.